rmf_demos_gz/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Changelog for package rmf_demos
+## Changelog for package rmf_demos_gz
```

Comment:
This is a new package and shouldn't have any prior history in its changelogs (same with `rmf_demos_ign`). When we update the version/changelogs for the `galactic` release we can append the log with https://github.com/open-rmf/rmf_demos/pull/78 and this PR